beta/azure-container-apps/ReadMe.md

```diff
@@ -0,0 +1,228 @@
+    ```
+    ResourceGroup="op-scim-bridge-rg"
+    Location="canadacentral"
+    ContainerAppEnvironment="op-scim-bridge-con-app-env"
+    ContainerAppName="op-scim-bridge-con-app"
+    ```
+3. Create the resource group 
+    ```
+    az group create --name $ResourceGroup --location $Location
+    ```
+4. Create the Container App Environment 
+    ```
+    az containerapp env create --name $ContainerAppEnvironment --resource-group $ResourceGroup --location $Locatio
+    ```
+5. Create the base Container App:
+    ```
+    az containerapp create -n $ContainerAppName -g $ResourceGroup --container-name op-scim-bridge \
+    --image docker.io/1password/scim:v2.8.2 --environment $ContainerAppEnvironment --ingress external \
+    --target-port 3002 --cpu 0.25 --memory 0.5Gi --min-replicas 1 --max-replicas 1 \
+    --env-vars 'OP_REDIS_URL=redis://localhost:6379' 'OP_SESSION=""'
+    ```
+    > **Note:** 
+    > 
+    >If you are prompted to enable the ContainerApp extension, with the error `The command requires the extension containerapp. Do you want to install it now? The command will continue to run after the extension is installed. (Y/n):`, type Y.
+
+6. Before you create the container Secret, upload your scimsession file to the Cloud Shell:
+   1. Click the “Upload/Download files” button and choose Upload.
+   2. Find the `scimsession` file that you saved to your computer and choose it.
+   3. Make note of the upload destination, then click Complete.
+7. Create Azure Container App Secret:
+    ```
+    az containerapp secret set -n $ContainerAppName -g $ResourceGroup --secrets scimsession="$(cat /home/$USER/scimsession | base64)"
+    ```
+8. Update your Container App with the required secondary container for Redis:
+    ```
+    az containerapp update -n $ContainerAppName -g $ResourceGroup --container-name op-scim-redis \
+    --image docker.io/redis --cpu 0.25 --memory 0.5Gi --set-env-vars 'REDIS_ARGS=--maxmemory 256mb --maxmemory-policy volatile-lru'
+    ```
+9. Update your Container App to allow your SCIM bridge container to use the secret you have created:
+    ```
+    az containerapp update -n $ContainerAppName -g $ResourceGroup --container-name op-scim-bridge \
+    --set-env-vars 'OP_REDIS_URL=redis://localhost:6379' 'OP_SESSION=secretref:scimsession'
+    ```
+10. Find your fdqn of your deployed application, this will be known as your SCIM bridge URL:
+    ```
+    az containerapp ingress show -n $ContainerAppName -g $ResourceGroup
+    ```
+11. Open the fqdn listed in a seperate browser tab, where you can test your connection. You should be prompted to log into the SCIM bridge with your `bearer token`.
+
+### Step 3: Follow the steps to connect your Identity provider to the SCIM bridge.
+ - [Connect your Identity Provider](https://support.1password.com/scim/#step-3-connect-your-identity-provider)
+
+
+<hr>
+
+## Manual Azure Portal Deployment Steps
```

Comment:
I'm wondering whether we should be front-loading this README with the portal instructions. I assume that the majority of people who will be using this deployment method skew towards preferring the GUI. Front-loading this with the portal instructions may be less intimidating.

Reply:
Just to close the loop on this discussion. Since more examples in scim-examples are leading customers to command based deployments, we will prioritize this method to align with the rest of our examples. The ability to deploy with a few commands directly in the Azure Cloud Shell is a huge plus here.

Reply:
Agreed in light of our offline discussions.